src/components/Table/GridTable.stories.tsx

```diff
@@ -1085,3 +1086,53 @@ export function RevealOnRowHover() {
     </>
   );
 }
+
+export function Toggle_Custom_Collapse() {
```

Comment:
Nit suggestion: We don't typically put `_` in names.

Reply:
Done 👍 
Thanks